plugins/modules/zos_unarchive.py

```diff
@@ -157,7 +157,7 @@
     description:
       - List the directory and file or data set names that you would like to
         exclude from the unarchive action.
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```

Comment:
This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

Reply:
fixed